sheet_to_triples/trans.py

```diff
@@ -80,9 +80,13 @@ def __repr__(self):
         return f'<{self.__class__.__name__} {self.name!r}>'
 
     @classmethod
-    def iter_from_name(cls, name):
-        transforms_dir = os.getenv('TRANSFORMS_DIR', 'transforms')
-        path = os.path.join(transforms_dir, name + '.py')
+    def iter_from_name(cls, name, base_path=None):
+        if not base_path:
```

Comment:
This should be the `is None` spelling - an empty string is a relative path.

Reply:
Whoops, I already merged. Will stick up another PR.